demos/python_demos/human_pose_estimation_3d_demo/human_pose_estimation_3d_demo.py

```diff
@@ -0,0 +1,145 @@
+import json
+import os
+
+import cv2
+from argparse import ArgumentParser, SUPPRESS
+import numpy as np
+
+from modules.inference_engine_openvino import InferenceEngineOpenVINO
+from modules.input_reader import VideoReader, ImageReader
+from modules.draw import Plotter3d, draw_poses
+from modules.parse_persons import parse_poses
+
+
+def rotate_poses(poses_3d, R, t):
+    R_inv = np.linalg.inv(R)
+    for pose_id in range(len(poses_3d)):
+        pose_3d = poses_3d[pose_id].reshape((-1, 4)).transpose()
+        pose_3d[0:3, :] = np.dot(R_inv, pose_3d[0:3, :] - t)
+        poses_3d[pose_id] = pose_3d.transpose().reshape(-1)
+
+    return poses_3d
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(description='Lightweight 3d human pose estimation demo. '
+                                        'Press esc to exit, "p" to (un)pause video or process next image.',
+                            add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument('-m', '--model',
+                      help='Required. Path to an .xml file with a trained model '
+                           '(or .pth in case of PyTorch inference).',
+                      type=str, required=True, metavar='"<path>"')
+    args.add_argument('--video', help='Optional. Path to video file or camera id.', type=str, default='')
+    args.add_argument('-d', '--device',
+                      help='Optional. Specify the target device to infer on: CPU, GPU, FPGA, HDDL or MYRIAD. '
+                           'The demo will look for a suitable plugin for device specified '
+                           '(by default, it is CPU).',
+                      type=str, default='CPU', metavar='"<path>"')
+    args.add_argument('--use-pytorch',
+                      help='Optional. Run network with PyTorch as inference engine. '
+                           'Only CPU and GPU devices are supported.',
+                      action='store_true')
+    args.add_argument('--images', help='Optional. Path to input image(s).', nargs='+', default='')
+    args.add_argument('--height-size', help='Optional. Network input layer height size.', type=int, default=256)
+    args.add_argument('--extrinsics-path',
+                      help='Optional. Path to file with camera exstrinsics.',
+                      type=str, default=None, metavar='"<path>"')
+    args.add_argument('--fx', type=np.float32, default=-1, help='Optional. Camera focal length.')
+    args = parser.parse_args()
+
+    if args.video == '' and args.images == '':
+        raise ValueError('Either --video or --image has to be provided')
+
+    stride = 8
+    if not args.use_pytorch:
```

Comment:
There is no need to invert `args.use_pytorch` value before checking it because you have `else`, so you cover both values any way.